tests/pages/MeetingWidget.page.js

```diff
@@ -1,14 +1,22 @@
 class MeetingWidgetPage {
   get destination() { return $('input[placeholder="Widget Destination"]'); }
   get displayWidgetBtn() { return $('button[aria-label="Display Meeting Widget"]'); } // #md-button-3
+  get removeWidgetBtn() { return $('button[aria-label="Remove Meeting Widget"]'); } // #md-button-4
   get interstitialMeeting() { return $('.wxc-interstitial-meeting'); }
   get meetingInfo() { return $('.wxc-meeting-info'); }
 
+  get muteAudioBtn() { return $('.meeting-controls-container button[alt=Mute]'); }
+  get unmuteAudioBtn() { return $('.meeting-controls-container button[alt=Unmute]'); }
+
   loadWidget(meetingDestination) {
     this.destination.setValue(meetingDestination);
     this.displayWidgetBtn.click();
     this.interstitialMeeting.waitForDisplayed({timeout: 30000});
   }
+
+  unloadWidget() {
+    this.removeWidgetBtn.click();
+  }
```

Comment:
it would be nice to also wait for the widget's disappearance:
```js
this.meetingWidget.waitForExist({reverse: true, timeout: 3000});
```

`this.meetingWidget` can be `$('.meeting-widget')` and should be defined between `removeWidgetBtn` and `interstitialMeeting`